server/views/review/review.handlebars

```diff
@@ -0,0 +1,126 @@
+<body>
```

Comment:
can you remove the <body > its already there in main

Reply:
I think we should check our page source in html validator.

Reply:
Ok, I will correct them and push it again.